plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotIntegrationConnectorSmokeTest.java

```diff
@@ -1864,6 +1864,52 @@ public void testAggregationPushdown()
                 .isThrownBy(() -> query("SELECT bool_col, COUNT(long_col) FROM \"SELECT bool_col, long_col FROM " + ALL_TYPES_TABLE + " GROUP BY bool_col, long_col\""))
                 .withRootCauseInstanceOf(RuntimeException.class)
                 .withMessage("Operation not supported for DISTINCT aggregation function");
+
+        // Verify that count(<column name>) is pushed down only when it matches a COUNT(DISTINCT <column name>) query
+        assertThat(query("SELECT COUNT(bool_col) FROM" +
```

Comment:
use `"""`

Reply:
Noticed that using `formatted(...)` disables the SQL formatting so the table names are hardcoded. Seems to be similar to other use cases, lmk if you prefer using `formatted(tableName)`.